tests/integration/ovs_test.py

```diff
@@ -638,6 +645,69 @@ def test_remove_all_ovsdb_global_config():
     }
 
 
+def test_ovsdb_global_config_add_delete_mapping():
```

Comment:
Please add two more test cases:
 1. Delete one map out of two+ maps.
 2. Replace/Modify a map via:
```yml
ovn:
  bridge-mappings:
    - localnet: yellow
      state: absent
    - localnet: yellow
      bridge: ovsbr5
```

Reply:
I think I see; your objective is to assert we can
1. remove without naming the bridge
2. state is optional when adding / preserving

Right ?

Reply:
Wait, I just now noticed your second example features the **same** localnet key for both listed mappings.

That's weird... Is that intentional ? I really don't know what the expected behavior should be.
@cathay4t

Reply:
> 1. Delete one map out of two+ maps.

This is to make sure nmstate will not touch undesired ovn bridge map.

> 2. Replace/Modify a map via

This is for changing OVN mapping in single YML.